### ISSUE TITLE
Missing message in production (Vercel)

### ISSUE THREAD

**Issue by johnboylesingfield** [amannn/next-intl#161]
### Description

Disclaimer: Sorry in advance if this is not a bug and some mistake of mine. I've been trying to make a bilingual Nextjs app with firebase db/auth without having much experience of theses things before starting this project. I'm learning quickly!

Some missing message errors are showing in my production build on Vercel:

`
[...]
[11:59:39.310] info  - Collecting page data...
[11:59:39.938] info  - Generating static pages (0/18)
[11:59:39.971] n [Error]: MISSING_MESSAGE
[11:59:39.972]     at I (/vercel/path0/node_modules/use-intl/dist/use-intl.cjs.production.min.js:1:2838)
[11:59:39.972]     at /vercel/path0/node_modules/use-intl/dist/use-intl.cjs.production.min.js:1:8135
[11:59:39.973]     at Object.Kc [as useMemo] (/vercel/path0/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:60:240)
[11:59:39.973]     at Object.exports.useMemo (/vercel/path0/node_modules/react/cjs/react.production.min.js:25:208)
[11:59:39.973]     at /vercel/path0/node_modules/use-intl/dist/use-intl.cjs.production.min.js:1:8108
[11:59:39.974]     at exports.useTranslations (/vercel/path0/node_modules/use-intl/dist/use-intl.cjs.production.min.js:1:8396)
[11:59:39.974]     at Header (/vercel/path0/.next/server/pages/_app.js:140:73)
[11:59:39.974]     at Wc (/vercel/path0/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
[11:59:39.975]     at Zc (/vercel/path0/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
[11:59:39.975]     at Z (/vercel/path0/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89) {
[11:59:39.975]   code: 'MISSING_MESSAGE',
[11:59:39.976]   originalMessage: undefined
[11:59:39.976] }
[...]
`

It seems to me the problem is related to dynamic routes:
https://github.com/johnboylesingfield/appel-reviews/blob/main/pages/call/%5Bid%5D.js

The bug used to generate a 404 error on the live website when I was trying to access dynamic pages. I changed getstaticprops to getserversideprops and the 404s dissapeared (no idea why though). However, the missing messages in the logs remain.

I am also noticing a suspiciously long loading time for a first request on dynamic pages. It might be linked to another issue, but it is worth noting just in case.

Live site: 
https://www.appel.reviews/

### Reproduction

I don't know how to reproduce this bug in codesandbox, but here is my github repo:

https://github.com/johnboylesingfield/appel-reviews

https://www.appel.reviews/

### Expected behaviour

If possible, I would like the errors to dissapear and the loading time to improve to a normal level.

Thank you so much for your help.

**Comment by johnboylesingfield** [comment]
Update: after inspecting the network get requests, I can see that the json file containing the translations takes 1,8s to load, which seems way too long for a file as tiny as this:

https://www.appel.reviews/_next/data/fHSA8Q3c1xs-FdaBmD8bP/en/call/rM4WTMzmFkQlLX2Q4NmE.json?id=rM4WTMzmFkQlLX2Q4NmE

**Comment by johnboylesingfield** [comment]
I fixed the loading delay by adding getstaticprops (my bad!), but the error messages remains in Vercel's build logs. Should I ignore them?

**Comment by amannn** [comment]
Hi @johnboylesingfield and thank you so much for your kind sponsorship, it means a lot to me! ❤️ 

It seems like you were able to resolve the bug? Are you still seeing the error message in your logs?
